pkg/kustomize/kustomize.go

```diff
@@ -462,6 +462,7 @@ func Inflate(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistry, baseCo
 		}
 	}
 
+	// FIXME(alecmerdler): Should we create certs if `route` component is unmanaged...?
 	log.Info("Ensuring `ssl.cert` and `ssl.key` pair for Quay app TLS")
 	tlsCert, tlsKey, err := EnsureTLSFor(ctx, quay, baseConfigBundle.Data["ssl.cert"], baseConfigBundle.Data["ssl.key"])
```

Comment:
@dmesser Do you have opinions on whether we should generate self-signed certs for Quay when the `route` component is unmanaged?

Reply:
Having TLS termination at the Quay level (no edge termination) is much preferred.

Reply:
Unless the user provides customs certs thats ok. However in order for integration with Ingress and Routes to work we should expose these generated certs as `Secrets` (and link them on the CR) so users have a chance to set up a re-encrypting `Routes`: https://docs.openshift.com/container-platform/4.6/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes

Reply:
Yes, the suggestion will be to grab the self-signed certs from the Quay config bundle `Secret` and create a re-encrypt `Route`. That should satisfy everyone.